src/client/app/frameworks/i18n/components/lang-switcher.component.ts

```diff
@@ -3,11 +3,12 @@ import {Store} from '@ngrx/store';
 import 'rxjs/add/operator/take';
 
 // app
-import {FormComponent, Config, LogService, ILang} from '../../core/index';
+import {Config, LogService, ILang} from '../../core/index';
 import {ElectronEventService} from '../../electron/index';
 import {MultilingualService} from '../index';
+import {Component} from '@angular/core';
 
-@FormComponent({
+@Component({
```

Comment:
good call. i didnt know what would happen w/ the BaseComponent as NgModule seemed to replace some of the reason that existed, but was just a guess.